website/forms/php/helpers.md

```diff
@@ -0,0 +1,58 @@
+---
+id: helpers
+title: Helpers
+---
+
+Eightshift Forms offers static helpers that can be implemented in your project. The key difference between these helpers and filters is that you can use the former in your theme or plugin code regardless of the load cycle.
+
+### esFormsGetFormIdByName
+
+This function retrieves the custom unique name of a form as set in the settings when provided with the form ID. When applying specific settings to one or more forms using project filters, it is handy. Usually, forms are identified by their IDs, but if you switch the environment, the form ID may change. This helper helps you identify the form by its name instead of its ID.
```

Comment:
```suggestion
This function retrieves the custom, unique name of a form, as set in the Form settings, from the provided form ID. It's useful when applying specific settings to one or more forms using filters in your project.

 Form identification by name (instead of the default "by ID"), can be useful if switching environments when the form ID could change.
```